### DISCUSSION TITLE
能否支持toc默认pin到页面上，不需要鼠标hover到上面才展示呢？

### DISCUSSION THREAD

**Discussion by xbpk3t** [jaywcjlove/reference#310] [Ideas]
有的文档比如docker或者redis的，内容很多，查找起来就不太方便，如果可以优化一下toc，使用时会方便很多

![image](https://user-images.githubusercontent.com/8591495/220568758-54895ae8-431a-43ae-af18-f2cad4893917.png)

**Comment by jaywcjlove** [comment]
@hapihacking 目前没有更美观的界面设计，所以不知道更改成什么样子

**Comment by xbpk3t** [comment]
如果能支持toc默认pin到页面上（比如下面几个文档），对我来说就足够了，现在页面就挺不错的

![image](https://user-images.githubusercontent.com/8591495/220572009-08da42d3-6fd2-430f-9a9f-310de6ad0ff5.png)

![image](https://user-images.githubusercontent.com/8591495/220572146-3e09cdb6-6620-4cee-9fd5-e93b1559fcb9.png)

![image](https://user-images.githubusercontent.com/8591495/220572367-0b94481a-89c2-47ed-8892-2cf79a4bec70.png)

**Comment by jaywcjlove** [comment]
贴在卡片页面很难看 @hapihacking 搞过了，改成现在这样的

**Comment by jaywcjlove** [comment]
<img width="1235" alt="image" src="https://user-images.githubusercontent.com/1680273/220573074-3dcdfc8d-7011-40ae-bc02-d25515803ee3.png">

旁边不够的时候还有大片空白

**Comment by xbpk3t** [comment]
okk
